sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java

```diff
@@ -94,6 +94,6 @@ public String getVersion() {
      * @return the latest {@link BlobServiceVersion}
      */
     public static BlobServiceVersion getLatest() {
-        return V2021_06_08;
+        return V2021_08_06;
```

Comment:
I think we also need to update this in Share File and any other storage package that has a `*ServiceVersion.java` file.

Reply:
don't forget about https://github.com/Azure/azure-sdk-for-java/blob/5087bf2173419f89f7e381d267f6d2b8b7de750f/sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java#L90-L91

Reply:
and https://github.com/Azure/azure-sdk-for-java/blob/5087bf2173419f89f7e381d267f6d2b8b7de750f/sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/Constants.java#L219